framework/Shell/Actions/TPphShellAction.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * TPphShellAction class file
+ *
+ * @author Brad Anderson <belisoful@icloud.com>
+ * @link https://github.com/pradosoft/prado
+ * @license https://github.com/pradosoft/prado/blob/master/LICENSE
+ * @package Prado\Shell\Actions
+ */
+
+namespace Prado\Shell\Actions;
+
+use Prado\Prado;
+use Prado\Shell\TShellAction;
+
+/**
+ * Creates and run a Prado application in a PHP Shell.
+ *
+ * @author Wei Zhuo <weizhuo[at]gmail[dot]com>
+ * @package Prado\Shell\Actions
+ * @since 3.0.5
+ */
+class TPphShellAction extends TShellAction
```

Comment:
TPphShellAction should probably be TPhpShellAction (typo)

Reply:
indeed. I changed it on one version but it wasn't the version that was checked in.  Changing now.